examples/v1beta1/sdk/tune-train-from-func.ipynb

```diff
@@ -422,7 +463,10 @@
    "cell_type": "markdown",
    "id": "1ee6b88d-6dfa-4924-8c67-86e019c78012",
    "metadata": {
-    "tags": []
+    "tags": [],
+    "pycharm": {
+     "name": "#%% md\n"
+    }
```

Comment:
I would suggest to edit these Notebooks using JupyterLab directly.
In that case, the JSON format will be correctly rendered for every IDE.
E.g. you can just run JupyterLab locally to edit them:
```
pip install jupyterlab
jupyter lab
```